docs/rl-algorithms/ppo.md

```diff
@@ -712,6 +716,57 @@ Tracked experiments and game play videos:
+```
+
+### Explanation of the logged metrics
+
+See [related docs](/rl-algorithms/ppo/#explanation-of-the-logged-metrics) for `ppo.py`. Just like in [ppo_atari_envpool_xla_jax.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_atari_envpool_xla_jax.py), we omit logging `losses/old_approx_kl` and `losses/clipfrac` for brevity. The metrics are the same as those in [ppo_atari_envpool_xla_jax.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_atari_envpool_xla_jax.py).
+
+### Implementation details
+
+[ppo_atari_envpool_xla_jax_scan.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_atari_envpool_xla_jax_scan.py) is a clone of [ppo_atari_envpool_xla_jax.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_atari_envpool_xla_jax.py) that replaces the python loops with native `jax.scan`.
+
+### Experiment results
+
+To run benchmark experiments, see :material-github: [benchmark/ppo.sh](https://github.com/vwxyzjn/cleanrl/blob/master/benchmark/ppo.sh). Specifically, execute the following command:
+
+<script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fvwxyzjn%2Fcleanrl%2Fblob%2Fmaster%2Fbenchmark%2Fppo.sh%23L108-L113&style=github&type=code&showBorder=on&showLineNumbers=on&showFileMeta=on&showFullPath=on&showCopy=on"></script>
+
+Below are the average episodic returns for `ppo_atari_envpool_xla_jax_scan.py` in 3 atari games. It has the same sample efficiency as `ppo_atari_envpool_xla_jax.py`.
+
+|              | ppo_atari_envpool_xla_jax_scan ({'tag': ['pr-328']})   | baselines-ppo2-cnn ({})   | ppo_atari_envpool_xla_jax_truncation ({})   |
+|:-------------|:-------------------------------------------------------|:--------------------------|:--------------------------------------------|
+| BeamRider-v5 | 2899.62 ± 482.12                                       | 2835.71 ± 387.92          | 3133.78 ± 293.02                            |
+| Breakout-v5  | 451.27 ± 45.52                                         | 405.73 ± 11.47            | 465.90 ± 14.30                              |
+| Pong-v5      | 20.37 ± 0.20                                           | 20.45 ± 0.81              | 20.62 ± 0.18                                |
+
+???+ info
+
+    The speed of this variant and [ppo_atari_envpool_xla_jax.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_atari_envpool_xla_jax.py) are very similar but the compilation time is reduced significantly (see [vwxyzjn/cleanrl#328](https://github.com/vwxyzjn/cleanrl/pull/328#issuecomment-1340474894)). In the following learning curve, the speed increase comes from the fact that better hardware were used.
+
+Learning curves:
+
+![](../ppo/ppo_atari_envpool_xla_jax_scan/compare.png)
+
+Tracked experiments:
+
+<iframe src="https://wandb.ai/51616/cleanrl/reports/Regression-Report-ppo_atari_envpool_xla_jax_truncation--VmlldzozMTg4NzI3" style="width:100%; height:500px" title="Atari-CleanRL-s-PPO-Envpool-Jax-scan"></iframe>
+
```

Comment:
You might also want to regenerate this link, then clone the report under the `openrlbenchmark/openrlbenchmark` name space.

<img width="1324" alt="image" src="https://user-images.githubusercontent.com/5555347/208780242-86a48552-60eb-4c5a-9350-aa7a3267dfe0.png">

Reply:
updated